Misc/NEWS.d/next/Library/2022-08-14-18-59-54.gh-issue-69142.6is5Pq.rst

```diff
@@ -0,0 +1 @@
+add %:z strftime format code (generates tzoffset with colons as separator)
```

Comment:
```suggestion
Add ``%:z`` format code support to :meth:`~datetime.date.strftime` (generates tzoffset with colons as separator).
```

Reply:
this is not just `date`, it is used at misc. places.